### DISCUSSION TITLE
(Mini.files) Can I use `j/k` to confirm my changes?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvim-mini/mini.nvim#584] [Q&A] [answered]
![image](https://github.com/echasnovski/mini.nvim/assets/95092244/121a3c99-b51c-4318-ad6f-80d93842c62d)

`Y/N` is too faraway to press

**Comment by echasnovski** [comment, marked as answer]
Sorry, no, that is too much customization for what seems to be quite common interface.

To be clear:
- This message is not for change confirmation, but when you exit 'mini.files' (by default with `q`) when there is at least one modified buffer (meaning there are changes and after pressing `y` they will be forgotten). To confirm changes, press `=` (by default).
- The default choice is marked with `[]` brackets. So you don't have to press `Y` to close it without synchronization, `<CR>` (`<Enter>`) also works.
- It shouldn't be capital `Y` and `N`. Lower case `y` and `n` works too.
